util/ci-run-file-creator.py

```diff
@@ -159,7 +159,7 @@ def update_endpoint_sub_type():
     run_file["tags"]["endpoint-sub-type"] = args.endpoint_sub_type
 
     match args.endpoint:
-        case "remotehost":
+        case "remotehost"|"remotehosts":
```

Comment:
could your please refresh me on the needs of both?

Reply:
Both `remotehost` and `remotehosts`?  This script is being used to convert templates into usable JSON run-files for all endpoints.  This particular function is currently only doing something meaningful for the `k8s` endpoint so both `remotehost` and `remotehosts` are lumped together in what amounts to a `noop` case.